fingers: [finger1, finger2]
finger1:
  motor_index: 1
  control_topic: /barrett_hand/finger_1_joint1_position_controller/command
  frame: rfdistal
  sensor:
    rate: 100
    device: /dev/comedi0
    calibration: /calibration/FT11585.cal
  contact:
    a: 9.0
    b: 9.0
    c: 9.0
    y00: 0.0
    z0: 11.0
    frot: !degrees -18.0
    lm:
      write_to_file: false
      max_iter: 100
      tau: 1e-3
      e1: 1e-4
      e2: 1e-4
      e3: 1e-4
finger2:
  motor_index: 4
  control_topic: /barrett_hand/finger_2_joint1_position_controller/command
  frame: thdistal
  sensor:
    rate: 100
    device: /dev/comedi1
    calibration: /calibration/FT7486.cal
  contact:
    a: 9.0
    b: 9.0
    c: 9.0
    y00: 0.0
    z0: 11.0
    frot: !degrees -18.0
    lm:
      write_to_file: false
      max_iter: 100
      tau: 1e-3
      e1: 1e-4
      e2: 1e-4
      e3: 1e-4

